Makefile

```diff
@@ -45,11 +45,11 @@ clean:  ## clean all build, python, and testing files
 	rm -fr htmlcov/
 	rm -fr .pytest_cache
 
-build: ## run tox / run tests and lint
-	tox
+build: ## run tox / run tests and lint for local IO only
+	tox -- -k "not REMOTE"
```

Comment:
this just looks weird but I would imagine it is easier to negate the REMOTE than to add it in the other case?

Reply:
uhhhhhhhhh not sure. Will give it a try later this week.

Reply:
Just gave it a try. `"not REMOTE"` works best.